rero_ils/modules/selfcheck/api.py

```diff
@@ -306,13 +306,13 @@ def item_information(patron_barcode, item_pid, **kwargs):
             return item_information
 
 
-def selfcheck_checkout(transaction_user_pid, library_pid, patron_barcode,
+def selfcheck_checkout(transaction_user_pid, institution_id, patron_barcode,
```

Comment:
Can not understand why you need ` institution_id` as parameter. It is not used in the function.

Reply:
Yes, I know but it's the signature defined in invenio-sip2 module.
I will change that and use kwargs